dependencies.props

```diff
@@ -5,14 +5,14 @@
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json;
-      https://api.nuget.org/v3/index.json;
```

Comment:
@wtgodbe, is the feed url change expected?

Reply:
Yes